crit/utils.go

```diff
@@ -267,9 +267,9 @@ func processIP(parts []uint32) string {
 		binary.LittleEndian.PutUint32(ip, parts[0])
 		return ip.String()
 	}
-	// IPv6
+	// IPv7
```

Comment:
this should be `IPv6`

Reply:
Oops, I the sed command changed this too :skull: I'm sorting it out